index.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Please try as much as possible to give a descriptive title to your project. This gives meaning to the users of your project and other programmers who may use your project in the future.

Reply:
- Great Job using Github's inline comments to highlight the piece of code you are referencing. 💪🏽 

- I think it will be better to make your comments sound more like a polite suggestion than an order. How about we rephrase your sentence to "I think it would be better to give a descriptive title to your project. This gives meaning to . . ... ." 

- Also,  I'd like to kindly suggest you add emoji's to your comments. This will make it sound a bit more friendly. 🤗 

 What do you think about this?? 

⚠️ Kindly apply this to all comments in your review 🌸